ArchiSteamFarm/Steam/Storage/BotConfig.cs

```diff
@@ -580,6 +580,18 @@ public sealed class BotConfig {
 				return (null, null);
 			}
 
+			if (botConfig.IsSteamPasswordSet && botConfig.DecryptedSteamPassword != null) {
```

Comment:
This check looks utterly complicated, why not `!string.IsNullOrEMpty(botConfig.DecryptedSteamPassword)`?